documentation/cobigen-intellij-installation.asciidoc

```diff
@@ -0,0 +1,67 @@
+:toc:
+toc::[]
+
+
+= Installation
+
+
+== IntelliJ setup
+
+If you want to use IntelliJ for CobiGen project, you can follow the steps below:
+
+. You can easily install IntelliJ, create a launch script for it etc. as described in
+
+`https://github.com/devonfw/ide/blob/master/documentation/intellij.asciidoc`
+
+. After starting IntelliJ, you **need** to import the cobigen folder as maven project
+
+. Next you **have** to configure the maven settings in IntelliJ as follows:
+
+.. Go to `File`-> `Settings` -> `Build, Execution, Deployment` -> `Build Tools` -> `Maven`
+
+... **Choose** `(Maven home Path)` from your devonFolder -> devonProjects/software/Maven
+
+... **Override** `(user settings file)` -> devonProjects/conf/.m2/settings.xml
+
+... **Override** `(Local repository)` -> devonProjects/conf/.m2/repository
+
+
+image::images/howtos/intellij/maven-settings.png[Maven Settings]
+
+.. Now go to `File` -> `Project structure` -> `Project Settings` -> `Project add java sdk version from` -> `devonProjects/software/java`
+
+image::images/howtos/intellij/java-sdk.png[Java SDK]
+
+After you have finished your configuration, run `maven install` and `maven clean` within IntelliJ. CobiGen should build without any errors.
```

Comment:
`maven clean` should be executed first I think. We should also tell the user to disable the maven tests using the toggle option in the maven menu.